server/src/main/java/org/opensearch/gateway/remote/RemoteClusterStateService.java

```diff
@@ -790,6 +670,26 @@ public RemoteClusterStateManifestInfo markLastStateAsCommitted(ClusterState clus
         return committedManifestDetails;
     }
 
+    /**
+     * Fetch latest ClusterMetadataManifest from remote state store
+     *
+     * @param clusterUUID uuid of cluster state to refer to in remote
+     * @param clusterName name of the cluster
+     * @return ClusterMetadataManifest
+     */
+    public Optional<ClusterMetadataManifest> getLatestClusterMetadataManifest(String clusterName, String clusterUUID) {
+        return remoteManifestManager.getLatestClusterMetadataManifest(clusterName, clusterUUID);
+    }
+
+    public Optional<ClusterMetadataManifest> getClusterMetadataManifestByTermVersion(
```

Comment:
This method is not required. We will be sending the full path of manifest during publication. The follower node will download using the manifest path rather than the term and version.

Reply:
Removed